Assets/Plugins/iOS/Shopify/BuyTests/PaymentSessionTests.swift

```diff
@@ -68,7 +68,7 @@ class PaymentSessionTests: XCTestCase {
     //  MARK: - Supported Networks -
     //
     func testSupportedNetworks() {
-        XCTAssertEqual(PaymentSession.supportedNetworks, [.amex, .masterCard, .visa, .discover])
```

Comment:
Is changing the supported networks actually related to this change? (Also, won't this be a dynamic list soon enough anyways?)